src/libraries/tests.proj

```diff
@@ -21,7 +21,10 @@
     <!-- We currently only test with C# projects. -->
     <ProjectReference Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj"
                       Exclude="@(ProjectExclusions)"
-                      Condition="'$(BuildAllConfigurations)' != 'true'" />
+                      Condition="'$(BuildAllConfigurations)' != 'true' and ('$(TargetsMobile)' != 'true' and '$(TargetsBrowser)' != 'true')" />
```

Comment:
Let's leave this as it was as part of this PR. We can adjust later on the iOS/Android PRs.